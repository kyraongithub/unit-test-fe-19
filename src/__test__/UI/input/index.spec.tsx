import Input from '@/components/UI/input';
import { render } from '@testing-library/react';

describe('input component', () => {
  it('render input text', () => {
    const { container } = render(<Input type="text" value="" />);
    expect(container).toMatchSnapshot();
  });
  it('render input textarea', () => {
    const { container } = render(<Input type="textarea" value="" />);
    expect(container).toMatchSnapshot();
  });
});
