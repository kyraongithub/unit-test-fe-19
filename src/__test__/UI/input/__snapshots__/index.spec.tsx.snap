// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input component render input text 1`] = `
<div>
  <input
    type="text"
    value=""
  />
</div>
`;

exports[`input component render input textarea 1`] = `
<div>
  <textarea
    type="textarea"
  />
</div>
`;
